scripts/devops_tasks/common_tasks.py

```diff
@@ -25,7 +25,7 @@
     "mgmt",
     "azure-cognitiveservices",
     "azure-servicefabric",
-    "azure-nspkg",
+    "nspkg"
```

Comment:
Not azure-keyvault-nspkg?

Reply:
Correct. This allows any package with "nspkg" in its name to pass CI despite having no tests.